exercises/js/navigation.js

```diff
@@ -0,0 +1,34 @@
+function DropDownMenuHandler(menuDiv) {
+  this.menu = $(menuDiv);
+}
+
+DropDownMenuHandler.prototype.init = function() {
+  this.setMenuHover();
+};
+
+DropDownMenuHandler.prototype.setMenuHover = function() {
+  var _this = this;
+  this.menu.find('li').hover(function(event) {
+      var hoveredMenu = $(event.target);
+      // Change color of hovered item
+      hoveredMenu.addClass('hover');
```

Comment:
extract different functions for both.
Also, both functions are almost same, can't we use `toggleClass` here.